<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportVectorsDialog</class>
 <widget class="QDialog" name="ExportVectorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Vectors</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Vector Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="_changeVectorList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>The source file for the vectors selected in this list will be changed.</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_addAll">
            <property name="whatsThis">
             <string>Select all vectors in the list.</string>
            </property>
            <property name="text">
             <string>Add A&amp;ll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_add">
            <property name="whatsThis">
             <string>Select all vectors in the list.</string>
            </property>
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_remove">
            <property name="whatsThis">
             <string>Unselect all vectors in the list.</string>
            </property>
            <property name="text">
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_removeAll">
            <property name="whatsThis">
             <string>Unselect all vectors in the list.</string>
            </property>
            <property name="text">
             <string>R&amp;emove All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>51</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="_selectedVectorList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>The source file for the vectors selected in this list will be changed.</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_saveLocationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Save location:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Kst::FileRequester" name="_saveLocation" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::FileRequester</class>
   <extends>QWidget</extends>
   <header>filerequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportVectorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportVectorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
